src/main.rs

```diff
@@ -2339,17 +2339,9 @@ fn cmd_aggregate(
             let url_string = url.to_string();
             let audit_bytes = network.download(url).await?;
             let audit_string = String::from_utf8(audit_bytes).map_err(LoadTomlError::from)?;
-            let audit_source = SourceFile::new(&url_string, audit_string.clone());
-            let audit_file: AuditsFile = toml::de::from_str(&audit_string)
-                .map_err(|error| {
-                    let (line, col) = error.line_col().unwrap_or((0, 0));
-                    TomlParseError {
-                        source_code: audit_source,
-                        span: SourceOffset::from_location(&audit_string, line + 1, col + 1),
-                        error,
-                    }
-                })
-                .map_err(LoadTomlError::from)?;
+            let audit_source = SourceFile::new(&url_string, audit_string);
+            let audit_file =
+                storage::foreign_audit_source_to_local_warn(&url_string, audit_source)?;
```

Comment:
Maybe precede this call with a short comment describing why we're doing this here.